.travis.yml

```diff
@@ -52,7 +51,7 @@ anchors:
   envs: &common-job-env
     # Do not fall back to cache for "master" for PR on "release" branch:
     # adds the target branch to the cache key.
-    FOR_TRAVIS_CACHE=$TRAVIS_BRANCH
+    FOR_TRAVIS_CACHE=v1-$TRAVIS_BRANCH
```

Comment:
what does "v1" mean?

Reply:
"version 1" - I've used it to bump/discard the existing cache(s).